example/package.json

```diff
@@ -18,12 +18,11 @@
     "@types/node": "link:../node_modules/@types/node",
     "@types/react": "link:../node_modules/@types/react",
     "@types/react-dom": "link:../node_modules/@types/react-dom",
+    "ditto-react": "link:..",
     "react": "link:../node_modules/react",
     "react-dom": "link:../node_modules/react-dom",
     "react-scripts": "link:../node_modules/react-scripts",
-    "typescript": "link:../node_modules/typescript",
-    "ditto-react": "link:..",
-    "@dittowords/cli": "2.0.0"
+    "typescript": "link:../node_modules/typescript"
   },
   "devDependencies": {
     "@babel/plugin-syntax-object-rest-spread": "^7.8.3"
```

Comment:
I noticed that the example app's `yarn.lock` got deleted, but it looks like there's this random dev dependency left behind -- should we remove it as well / move it up to the root `package.json`?